### ISSUE TITLE
ebay-kleinanzeigen.de active thumbnail

### ISSUE THREAD

**Issue by auipga** [uBlockOrigin/uAssets#14459]
# TLDR Solution:
```
www.ebay-kleinanzeigen.de#@##viewad-lightbox-thumbnail-list > .ad-active
```
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [ ] uBO is the only extension
- [ ] uBO with default lists/settings
- [ ] using a new, unmodified browser profile

### URL(s) where the issue occurs

e.g. `https://www.ebay-kleinanzeigen.de/s-anzeige/schreibtinte-kwz-sheen-machine/2187776745-281-4266`

### Describe the issue

The active image thumbnail of the opened gallery view isn't visible.

### Screenshot(s)
**look at the green thumbnail at the bottom left**
uBlock disabled:
![image](https://user-images.githubusercontent.com/876689/185761949-7e93c249-0931-4bcb-b82e-2e944d821f98.png)

uBlock enabled:
![image](https://user-images.githubusercontent.com/876689/185761991-601873fd-b8ff-45da-9f19-8c03cc4f9cd7.png)

### Configuration

<details>

```yaml
uBlock Origin: 1.44.0
Firefox: 103
filterset (summary): 
  network: 103734
  cosmetic: 188491
  scriptlet: 24817
  html: 735
listset (total-discarded, last updated): 
  added: 
    https://raw.githubusercontent.com/Spam404/lists/master/adblock-list.txt: 7252-5, 1d.6h.9m
    adguard-annoyance: 56494-4802, 1d.6h.14m
    fanboy-thirdparty_social: 68-0, 5d.20h.52m
    fanboy-annoyance: 78871-63009, 5d.20h.51m
    fanboy-cookiemonster: 39545-131, 5d.20h.50m
    fanboy-social: 23293-28, 1d.6h.13m
    DEU-0: 5905-2, 5h.20m
    adguard-mobile: 8154-156, 1d.6h.12m
    ublock-annoyances: 4532-4, 1d.6h.11m
    adguard-social: 18230-1061, 1d.6h.10m
  default: 
    user-filters: 166-0, never
    ublock-filters: 33222-113, 1d.6h.19m
    ublock-badware: 4247-0, 1d.6h.18m
    ublock-privacy: 247-0, 13d.2m
    ublock-abuse: 76-0, 13d.1m
    ublock-unbreak: 1842-0, 1d.6h.17m
    easylist: 69593-767, 1d.6h.16m
    easyprivacy: 27390-600, 1d.6h.15m
    plowe-0: 3634-3, 13d.3m
    urlhaus-1: 5443-0, 5h.19m
    ublock-quick-fixes: 425-0, 5h.18m
filterset (user): [array of 170 redacted]
trustedset: 
  added: [array of 22 redacted]
  removed: 
    about-scheme
    edge-scheme
switchRuleset: 
  added: [array of 2 redacted]
urlRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: 
  cloudStorageEnabled: true
  contextMenuEnabled: false
  webrtcIPAddressHidden: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 5089 ms (selfie)
  maxAssetCacheWait: 1067 ms
```
</details>

**Comment by okiehsch** [comment]
`ebay-kleinanzeigen.de#@#.ad-active`
@khrin

**Comment by auipga** [comment]
IIRC there are other `.ad-active` elements, which shouldn't be excluded. That's why I prefixed with `#viewad-lightbox-thumbnail-list`

**Comment by Khrin** [comment]
@auipga May you open a new issue at https://github.com/easylist/easylistgermany/issues with the relevant link(s) where `##.ad-active` is supposed to be used? Thank you.

**Comment by VetaLioSTV** [comment]
Still same!
